modules/sdk-lib-mpc/src/tss/ecdsa-dkls/types.ts

```diff
@@ -104,7 +104,7 @@ export function deserializeP2PMessage(message: SerializedP2PMessage): Deserializ
     to: message.to,
     from: message.from,
     payload: new Uint8Array(Buffer.from(message.payload, 'base64')),
-    commitment: message.commitment ? new Uint8Array(Buffer.from(message.commitment, 'base64')) : undefined,
+    commitment: message.commitment ? new Uint8Array(Buffer.from(message.commitment, 'hex')) : undefined,
```

Comment:
The way HSM accepts the `commitment`, `hex` helps it to be fixed length and size-wise it doesn't hurt much since it's only 32 bytes.  `base64` helps keep the size of other messages shorter.